docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -3,99 +3,106 @@ Google Cloud Platform Guide
 * ``service_account_email``: email associated with the project
-* ``credentials_file``: path to the JSON credentials file
-* ``project_id``: id of the project
+* ``service_account_file``: path to the JSON credentials file
+* ``project``: id of the project
+* ``scopes``: The specific scopes that you want the actions to use.
 
-For example, to create a new instance using the cloud module, you can use the following configuration:
+For example, to create a new IP address using the `gcp_compute_address` module,
+you can use the following configuration:
 
 .. code-block:: yaml
 
-   - name: Create instance(s)
+   - name: Create IP address
      hosts: localhost
      connection: local
      gather_facts: no
 
      vars:
-       service_account_email: unique-id@developer.gserviceaccount.com
-       credentials_file: /path/to/project.json
-       project_id: project-id
-       machine_type: n1-standard-1
-       image: debian-7
+       service_account_file: /home/my_account.json
+       project: my-project
+       auth_kind: serviceaccount
+       scopes:
+         - www.googleapis.com/auth/compute
 
      tasks:
 
-      - name: Launch instances
-        gce:
-            instance_names: dev
-            machine_type: "{{ machine_type }}"
-            image: "{{ image }}"
-            service_account_email: "{{ service_account_email }}"
-            credentials_file: "{{ credentials_file }}"
-            project_id: "{{ project_id }}"
-
-When running Ansible inside a GCE VM you can use the service account credentials from the local metadata server by
-setting both ``service_account_email`` and ``credentials_file`` to a blank string.
-
-Configuring Modules with secrets.py
-```````````````````````````````````
-
-Create a file ``secrets.py`` looking like following, and put it in some folder which is in your ``$PYTHONPATH``:
-
-.. code-block:: python
-
-    GCE_PARAMS = ('i...@project.googleusercontent.com', '/path/to/project.json')
-    GCE_KEYWORD_PARAMS = {'project': 'project_id', 'datacenter': 'gce_zone'}
-
-Ensure to enter the email address from the created services account and not the one from your main account.
-
-Now the modules can be used as above, but the account information can be omitted.
-
-If you are running Ansible from inside a GCE VM with an authorized service account you can set the email address and
-credentials path as follows so that get automatically picked up:
-
-.. code-block:: python
-
-    GCE_PARAMS = ('', '')
-    GCE_KEYWORD_PARAMS = {'project': 'project_id', 'datacenter': ''}
+      - name: Allocate an IP Address
+        gcp_compute_address:
+            state: present
+            name: 'test-address1'
+            region: 'us-west1'
+            project: "{{ project }}"
+            auth_kind: "{{ auth_kind }}"
+            service_account_file: "{{ service_account_file }}"
+            scopes: "{{ scopes }}"
 
 Configuring Modules with Environment Variables
```

Comment:
Again, this will be stronger if you use the same wording you used above, so this section would be "Providing Credentials as Environment Variables"

Reply:
Done.